src/decoder.rs

```diff
@@ -1174,7 +1180,10 @@ fn compute_image(
     }
 }
 
-#[cfg(feature = "rayon")]
+#[cfg(all(
+    not(any(target_arch = "asmjs", target_arch = "wasm32")),
+    feature = "rayon"
+))]
```

Comment:
Could you move this implementation into `worker/rayon.rs`? That would isolate the feature requirement by source location. Also allows fixing the glob import `rayon::prelude::*` and turn it into a proper one.

Reply:
Is it ok what I did in https://github.com/image-rs/jpeg-decoder/pull/240/commits/7d843f15a19d3354cdf230243b3051dec785b100 ?

Reply:
Yes :+1: